app/jobs/mets_directory_scan_job.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# Very simple job.  Scan needs to occur in a job so that the mounts are available
+class MetsDirectoryScanJob < ApplicationJob
+  queue_as :default
+
+  def default_priority
+    100
```

Comment:
I think lower priority happen faster - is having this essentially at the bottom of our priority list intentional?